.github/workflows/test.yml

```diff
@@ -35,6 +35,9 @@ jobs:
     - name: Run isort
       run: isort --check-only --diff ./vyper ./tests ./setup.py
 
+    - name: Run mypy
```

Comment:
Can't we reuse the pre-commit config in this job? e.g. `pre-commit run --all-files`

Reply:
no, the pre-commit rule doesn't run `check-only`